openshift/fluentd.bc.yaml

```diff
@@ -0,0 +1,94 @@
+---
+apiVersion: v1
+kind: Template
+labels:
+  build: ${APP_NAME}-${INSTANCE}-custom
+  template: ${APP_NAME}-${INSTANCE}-template-bc
+metadata:
+  name: ${APP_NAME}-${INSTANCE}-template-bc
+objects:
+  - apiVersion: v1
+    kind: ImageStream
+    metadata:
+      name: ${APP_NAME}-custom
+    spec:
+      lookupPolicy:
+        local: false
+
+  - apiVersion: v1
+    kind: BuildConfig
+    metadata:
+      name: ${APP_NAME}-${INSTANCE}-custom
+    spec:
+      completionDeadlineSeconds: 600
+      failedBuildsHistoryLimit: 3
+      nodeSelector:
+      output:
+        to:
+          kind: ImageStreamTag
+          name: "${APP_NAME}-custom:${INSTANCE}"
+      postCommit: {}
+      resources:
+       limits:
+          cpu: "${CPU_LIMIT}"
+          memory: "${MEMORY_LIMIT}"
+       requests:
+          cpu: "${CPU_REQUEST}"
+          memory: "${MEMORY_REQUEST}"
+      runPolicy: SerialLatestOnly
+      source:
+        type: Dockerfile
+        dockerfile: |-
+          FROM fluent/fluentd:v1.7-debian-1
```

Comment:
Two questions here:
1. Any particular reason why we're using the debian based one instead of alpine? If we can get away with installing plugins on fluentd on alpine platform, we'll be able to benefit from a significantly smaller container image size.
2. Why are we using an older version v1.7 instead of v1.11-x?

Check the available image tags here in their docs: https://hub.docker.com/r/fluent/fluentd/

Reply:
good point!. ..there was a reason.. i forget what it was.. will take a look on monday